src/App.js

```diff
@@ -1,25 +1,22 @@
-import logo from './logo.svg';
 import './App.css';
+import './pages/home/Home'
+import {Header} from "./components/header/Header";
+import { Fetch } from "./components/Fetch/Fetch";
 
 function App() {
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <>
+<Header></Header>
```

Comment:
Als het component geen children mee krijgt gestuurd, dan kun je het ook zo doen
```suggestion
<Header />
```